client/src/store/reducers/widgets/currentVeiw/index.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  FulfilledAction,
+  FETCH_LOGIN_FULFILLED,
+} from "../../../actions/loginPage";
+
+export type CurrentView = {
+  isLogin: boolean;
+};
+
+const initialState = {
+  isLogin: false,
+};
+
+export const currentView = (
+  state: CurrentView = initialState,
+  action: FulfilledAction
+) => {
+  switch (action?.type) {
```

Comment:
Редьюсер не может вызваться без экшена, поэтому можно сразу обратиться к `type`

```suggestion
  switch (action.type) {
```

Reply:
исправил